CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release Notes
 
-## 0.3.0 (unreleased)
+## 0.3.0 (11.03.2021)
```

Comment:
```suggestion
## 0.3.0 (12.03.2021)
```
😛

Reply:
that was what I originally wanted to put there lol